src/template.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Document Title -->
+    <title>Todo App</title>
+    <!-- Icons Link -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+
+<body>
+    <div class="container" id="container">
+        <div class="todo-title-box">
+            <h2 class="title">To-Do list
+            </h2>
+            <i class='material-icons'>&#xE627;</i>
+        </div>
+        <div class="todo-input-box">
+            <input type="text" placeholder="Add to your list..." class="input">
```

Comment:
- [ ] [Optional] 👍 You've made a good addition by adding the input field. However, there's a small adjustment needed to align it with the design:

- [Optional]  Please add some padding to the input field and remove the outline to match the design as closely as possible.

| Your | Design |
| --- | --- |
| ![image](https://github.com/GhausAliKhan/Todo-App/assets/55840999/ae67b00c-5eec-45a1-9d00-e588c25f35dd) | ![image](https://github.com/GhausAliKhan/Todo-App/assets/55840999/839957bf-e6e7-44e8-a976-f602633546e3) |


By adding the padding, your input field will have a more visually consistent appearance and better align with the overall design. Keep up the great work, and I'm excited to see your project refined with this improvement! 🌟🎨🚀

Reply:
Your Review is appreciated 😍 
I'll definitely add this change 💯